src/app/content/services/backend.service.ts

```diff
@@ -15,13 +15,17 @@ export interface IBackendService {
   removeItem(id: number): Observable<any>;
 }
 
-const baseUrl = `${window.location.protocol}//${window.location.hostname}:${window.location.port}${environment.baseHref}`;
-
 @Injectable()
 export class BackendService implements IBackendService {
   private headers: Headers;
+  private baseUrl: String;
 
-  constructor(private http: AuthHttp) { }
+  constructor(private http: AuthHttp) {
+    this.baseUrl = `${window.location.href}`.replace('/content', '');
```

Comment:
actually didn't mean to approve this PR. Thought it only removes unused imports, but then saw this, which I really think is a hacky solution. There should be a better way.